rllib/models/torch/model.py

```diff
@@ -96,25 +96,25 @@ class TorchRecurrentModel(RecurrentModel, nn.Module, TorchModelIO):
         ...
         ...     @property
         ...     def input_spec(self):
-        ...         return SpecDict(
+        ...         return ModelSpecDict(
         ...             {"obs": "batch time hidden"}, hidden=self.config.input_size
```

Comment:
fix the docstring: `{"obs": TensorShape("b,t,h")}` etc (using `b,t,h` for conciseness.

Reply:
dude you didn't do this :)